conans/client/generators/cmake_common.py

```diff
@@ -96,6 +106,11 @@ def cmake_dependencies(dependencies, build_type=""):
 set(CONAN_SHARED_LINKER_FLAGS{build_type} "{deps.sharedlinkflags} ${{CONAN_SHARED_LINKER_FLAGS{build_type}}}")
 set(CONAN_EXE_LINKER_FLAGS{build_type} "{deps.exelinkflags} ${{CONAN_EXE_LINKER_FLAGS{build_type}}}")
 set(CONAN_C_FLAGS{build_type} "{deps.cflags} ${{CONAN_C_FLAGS{build_type}}}")
+
+# Apple Frameworks
+conan_find_apple_frameworks(CONAN_FRAMEWORKS{build_type} CONAN_FRAMEWORKS_FOUND{build_type})
```

Comment:
I would do this call earlier before the first CONAN_LIBS declaration. It is not easy to see than it has been aggregated. Is it possible?

Reply:
The thing here is that you need CONAN_FRAMEWORKS defined before this call and then assign the values found to CONAN_LIBS. So first frameworks, then function call, and finally conan_libs.

However, many tests check the library values of CONAN_LIBS and didn't want to modify all the tests for this PR.

If you advise me to do it, I will do it without any problem.